openadapt/events.py

```diff
@@ -103,31 +134,38 @@ def make_parent_event(child, extra=None):
     return models.ActionEvent(**event_dict)
 
 
-# Set by_diff_distance=True to compute distance from mouse to screenshot diff
-# (computationally expensive but keeps more useful events)
-def merge_consecutive_mouse_move_events(events, by_diff_distance=False):
-    """Merge consecutive mouse move events into a single move event"""
+def merge_consecutive_mouse_move_events(
+    events: list, by_diff_distance: bool = False
+) -> list:
+    """Merge consecutive mouse move events into a single move event.
 
-    _all_slowdowns = []
+    Args:
+        events (list): The list of events to process.
+        by_diff_distance (bool): Whether to compute the distance from the mouse to
+          the screenshot diff. This is computationally expensive but keeps more useful
+          events. Default is False.
 
+    Returns:
+        list: The merged list of events.
 
-    def is_target_event(event, state):
-        return event.name == "move"
+    """
+    _all_slowdowns = []
 
+    def is_target_event(event: Any, state: Any) -> bool:
```

Comment:
`event: ActionEvent, state: dict[Str, Any]`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507